Lib9c/Model/Character/ArenaCharacter.cs

```diff
@@ -300,6 +296,73 @@ private static CharacterSheet.Row CharacterRow(int characterId, ArenaSimulatorSh
             }
         }
 
+        public void SetRuneV2(
```

Comment:
How about this?
- Add `Obsolete` attribute to `SetRune()`
- Rename `SetRune()` to `SetRuneV1()`
- Rename `SetRuneV2()` to `SetRune()`